### ISSUE TITLE
No reject button needed

### ISSUE THREAD

**Issue by maxwang7** [precup/turing-machine-sim#137]
DFAs and NFAs nodes can only have two states: accepting states or non accepting states. A rejecting state is redundant.

**Comment by precup** [comment]
Not really, considering accepting is a button, not a check box. Plus, a selection could either be entirely accepting, entirely not accepting, or neither.

**Comment by moonj** [comment]
Hm. I think this is a valid nitpick, and I think I'd actually prefer just having an accepting state, making "non-accepting" more clearly the default. It's honestly just the aesthetics of having a checkbox in that bar.

![screen shot 2015-05-08 at 10 05 37 pm](https://cloud.githubusercontent.com/assets/1813243/7548788/6702d6c4-f5ce-11e4-8acf-832cf26c4667.png)

It doesn't look too hot. I'm not 100% opposed to keeping it as a button so let's do that for now, but I'll try to find a better solution.
